src/clr-angular/layout/_variables.login.scss

```diff
@@ -1,4 +1,4 @@
-/*!
```

Comment:
Why is this one different? It's not loud so it's ok, just curious about the special treatment.

Reply:
Not sure what happened here. I'll remove it when I make the change for html comments.